src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,23 @@
+import React, { render, screen} from '@testing-library/react';
+import SearchResults from "../components/SearchResults";
+
+describe("Search", () => {
+    const validProps = {
+        searchResults: ["moon", "venus"]
+    };
+
+    it("renders search result component correctly", () => {
+        const { asFragment } = render(
+            <SearchResults searchResults={validProps.searchResults} />);
+            expect(asFragment()).toMatchSnapshot();
+        });
+
+        it("renders placeholder if no results returned", () => {
+            const validProps = [];
+            render(<SearchResults results={validProps} />);
+      
+            const placeholderMessage = screen.getByText("No results");
```

Comment:
Fixing the test as above and it will still fail, as you are not looking for the text that actually appears on screen. This should be
```suggestion
            const placeholderMessage = screen.getByText("Sorry, no results found. Try again.");
```